Makefile

```diff
@@ -0,0 +1,48 @@
+pip:
+	pip install -e .
+
+pip-development: pip
+	pip install -e.[development]
+
+pip-docs: pip-development
+	pip install -r docs/requirements.txt
+	pip install doc2dash
+
+pip-wheel:
+	pip install wheel
+
+bdist-wheel:
+	python setup.py sdist bdist_wheel
+
+docs:
```

Comment:
@tzaffi Is `make docs` intended to be repeatable?

Here's the output I see upon 2nd run:

```
~/dev/pyteal make-and-flake *1 !1 ?6 ───────────────────────────────────────────────────────────────────────────────────  pyteal 10:27:59 AM
❯ make docs
cd docs && \
	make html && \
	doc2dash --name pyteal --index-page index.html --online-redirect-url https://pyteal.readthedocs.io/en/ _build/html && \
	tar -czvf pyteal.docset.tar.gz pyteal.docset
Running Sphinx v4.5.0
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

The HTML pages are in _build/html.
Destination path "./pyteal.docset" already exists.
make: *** [docs] Error 17
```

For local development, I've typically run `make html` and left the `doc2dash` to the build.  Without having research further, it seems like the command can be made repeatable by deleting `docs/.pyteal.docset`.

Though I don't know enough off-hand to comfortably suggest deleting the dir vs for example, decomposing `docs` into 2 commands.

I can investigate further.  I'm raising the question in case you've got a strong feeling off-hand.  Let me know if you'd like me to look deeper.

Reply:
I don't have a strong opinion. @jasonpaulos - do you have any thoughts about this? Maybe this is an example of where we _shouldn't_ create a `make` command

Reply:
@tzaffi If there's no strong opinion, my default stance is to make the command idempotent.  If doing so feels onerous, then I'm open to backing out the `make` command.

Reply:
Not onerous. I'll just add a `rm -rf`...

Reply:
Ok, am just adding `rm -rf`...

Reply:
I think it's misleading to have this be `make docs`, since this does a very specific thing that no one should have to do manually: it builds the docs and packages it in an archival format that's uploaded as a build artifact during our CI runs.

So I'd suggest either renaming it `bundle-docs` or not putting it into the Makefile

Reply:
going with the `bundle-docs` approach